### DISCUSSION TITLE
Solicitação de dados para contratação do local da Python Brasil 2025

### DISCUSSION THREAD

**Discussion by renan-asantos** [apyb/comunidade#156] [Pergunta]
Oi pessoal,

Eu sou um dos Big Kahuna da Python Brasil 2025 e decidimos o local da próxima edição. Para fechar o contrato preciso dos dados abaixo:

1. Contrato social ou Estatuto Social
2. Cartão CNPJ
3. Procuração (se houver)
4. RG, CPF e comprovante de residência do representante legal

Depois de escrito o contrato precisaria saber o fluxo para pagarmos o sinal de 10% da locação para efetivamente fechar o contrato.

**Comment by cecivieira** [comment]
@anicelysantos

**Comment by cecivieira** [comment]
@naanadr qual é o link para o kickoff que fizemos recentemente? Tu poderia colocar aqui para que @renan-asantos assista?

@renan-asantos por favor, coloca a atual diretoria dentro do grupo do telegram/whatsapp da organização da pybr2025. Outra coisa, seria interessante que **sempre** que você tenha dúvida, pergunte diretamente a gestão atual da APyB. Assim como quando precisar usar dinheiro da Associação, como será para fechar esse contrato. Qual é o valor dele? Precisa de ajuda para revisar?

**Reply by renan-asantos** [reply]
Diretoria da APyB adicionada no grupo do telegram, alinhado de marcar vocês nas issues do nosso repositório de organização e continuar o assunto nas issues específicas

**Comment by cecivieira** [comment]
@renan-asantos sugerimos leitura deste manual:
https://apyb.python.org.br/associados/guias/como-organizar-eventos/

E posterior preenchimento do formulário solicitando apoio da Associação.

**Reply by naanadr** [reply]
@renan-asantos [onboarding](https://drive.google.com/file/d/11guz90FWsRVVQdlqa6XZcwvxs1lM8EbM/view?usp=sharing) que fizemos com os big kahunas da PySE e PyNE. Esse onboarding é complementar a leitura do manual.

**Reply by renan-asantos** [reply]
Muito obrigado pessoal!

**Reply by naanadr** [reply]
@jbiason @jailsonpj recomendo vcs darem uma olhada no onboarding que gravamos e no guia 😄

**Reply by jailsonpj** [reply]
Valeu pessoal

**Comment by anicelysantos** [comment]
@renan-asantos neste link estão os documentos solicitados: https://drive.google.com/drive/folders/172dyLrX_to7g2ooQBT-S2DMGv9MdEmfJ?usp=drive_link

Compartilhei com teu email.

**Reply by renan-asantos** [reply]
Muito obrigado Any!

**Reply by vbuxbaum** [reply]
Oi @anicelysantos ! Tudo joia?

Um fornecedor aqui da Python Sudeste em BH também pediu dados complementares (inscrição estadual, cartão CNPJ, etc). Solicitei acesso à essa pasta, acho que já vai ter tudo o que precisamos

Abraço!

cc: @cecivieira

**Reply by cecivieira** [reply]
@vbuxbaum qual teu e-mail?

**Reply by vbuxbaum** [reply]
vitorbuxbaum@gmail.com

**Comment by cecivieira** [comment]
@mawkee dei acesso à pasta para o e-mail do Caipyra. Por favor, avisa aqui na thread quando você terminar de usar a documentação. POr ter muitos dados pessoais, abrimos o acesso temporariamente.
